config/routes.rb

```diff
@@ -1,4 +1,5 @@
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
   root to: 'posts#index'
+  get '/comments' , to: 'comments#index'
```

Comment:
Remover espaço depois do '/comments'
```suggestion
  get '/comments', to: 'comments#index'
```